### ISSUE TITLE
"secret" is not exported from module 

### ISSUE THREAD

**Issue by wabiloo** [kazhala/InquirerPy#60]
I'm seeing this error reported by Pylance in VSCode.

Seems like you had it before, and closed it.
Is the version on PyPI behind?

**Comment by Gracecr** [comment]
Yes, #52 was merged in November 22, however the latest PyPi release is from June 2022. Until @kazhala has time to push a new release, you can install the master branch with `pip install git+https://github.com/kazhala/InquirerPy`.

**Comment by CanePlayz** [comment]
> Yes, #52 was merged in November 22, however the latest PyPi release is from June 2022. Until @kazhala has time to push a new release, you can install the master branch with `pip install git+https://github.com/kazhala/InquirerPy`.

Thanks
